third_party/blink/renderer/modules/BUILD.gn

```diff
@@ -167,6 +167,10 @@ component("modules") {
     "//third_party/blink/renderer/modules/xr",
   ]
 
+  if (is_cobalt) {
+    sub_modules += [ "//third_party/blink/renderer/modules/cobalt/crash_context" ]
```

Comment:
@yell0wd0g having the module name be cobalt in the codebase while still maintaining the H5vcc prefix of the interfaces would allow us to have a consistent codebase (our files are always in //cobalt and //starboard dirs) while still having the h5vcc api, but it does seem like a bad pattern to be naming them different things—wdyt?

Reply:
There's been a desire to rename the "h5vcc" namespace name, which refers to Cobalt's predecessor, to "cobalt" - maybe this is a good time to do it?

Reply:
Andrew suggested we discuss web API naming conventions with Kabuki during review of the h5vcc survey, which I think is a good idea. I'll keep this as is for now and possibly adjust later, based on the h5vcc review.